specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Line 570:
```
ballot_manifest: TYPE = set[ballot_manifest_info]
```
Should be a sequence? They want to be able to sort selections in order by the manifest, i.e. by "ascending order of Scanner/Batch/Position nos." (Audit Board Reporting Utility Mock-up.googledoc),

I note the database already has them in order in `ballot_manifest_table`, if that order is actually the same as Scanner/Batch/Position nos.